### PR TITLE
tidb: `tidb_service_scope` GA

### PR DESCRIPTION
**Pull request by hfxsd**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-distributed-execution-framework.md [hunk 1]
```diff
@@ -78,10 +78,6 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
 
 3. 从 v7.4.0 开始，你可以根据实际需求，调整执行后端任务的 TiDB 节点数量，在部署 TiDB 后为每一个 TiDB 节点设置 Instance 级别系统变量 [`tidb_service_scope`](/system-variables.md#tidb_service_scope-从-v740-版本开始引入)。`tidb_service_scope` 设置为 `background` 时，TiDB 节点可执行后端任务。`tidb_service_scope` 设置为默认值 "" 时，TiDB 节点不可执行后端任务。如果所有节点均未配置 `tidb_service_scope`，框架将调度所有节点执行后端任务。
```
Comment (ywqzzy) [inline tidb-distributed-execution-framework.md:79]:
```suggestion
3. 从 v7.4.0 开始，你可以根据实际需求，调整执行后端任务的 TiDB 节点数量，在部署 TiDB 后为每一个 TiDB 节点设置 Instance 级别系统变量 [`tidb_service_scope`](/system-variables.md#tidb_service_scope-从-v740-版本开始引入)。`tidb_service_scope` 设置为 `background` 时，TiDB 节点可执行后端任务。`tidb_service_scope` 设置为默认值 "" 时，TiDB 节点不可执行后端任务。如果所有节点均未配置 `tidb_service_scope`，框架将调度所有节点执行后端任务。
> **注意：**
>
> 在分布式任务执行过程中，如果某些 TiDB 节点下线后，分布式任务将随机分配子任务到剩余节点上。此时无法通过 `tidb_service_scope` 来动态均衡子任务的分配。
```

File: tidb-distributed-execution-framework.md [hunk 2]
```diff
@@ -78,9 +78,10 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
...
 
-    > **警告：**
-    >
-    > `tidb_service_scope` 目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+> **注意：**
+>
+> - 在分布式任务执行过程中，如果某些 TiDB 节点下线后，分布式任务将随机分配子任务到可用节点上，此时无法通过 `tidb_service_scope` 来动态均衡子任务的分配。
+> - 在分布式任务执行过程中，如果子任务均下发完毕，此时无法通过 `tidb_service_scope` 配置新节点执行该任务。
```
Comment (hfxsd) [inline tidb-distributed-execution-framework.md:84]:
```suggestion
    > **注意：**
    >
    > - 在分布式任务执行过程中，如果某些 TiDB 节点下线后，分布式任务将随机分配子任务到可用节点上，此时无法通过 `tidb_service_scope` 来动态均衡子任务的分配。
    > - 在分布式任务执行过程中，如果子任务均下发完毕，此时无法通过 `tidb_service_scope` 配置新节点执行该任务。
```

File: tidb-distributed-execution-framework.md [hunk 3]
```diff
@@ -78,9 +78,10 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
 
 3. 从 v7.4.0 开始，你可以根据实际需求，调整执行后端任务的 TiDB 节点数量，在部署 TiDB 后为每一个 TiDB 节点设置 Instance 级别系统变量 [`tidb_service_scope`](/system-variables.md#tidb_service_scope-从-v740-版本开始引入)。`tidb_service_scope` 设置为 `background` 时，TiDB 节点可执行后端任务。`tidb_service_scope` 设置为默认值 "" 时，TiDB 节点不可执行后端任务。如果所有节点均未配置 `tidb_service_scope`，框架将调度所有节点执行后端任务。
 
-    > **警告：**
+    > **注意：**
     >
-    > `tidb_service_scope` 目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+    > - 在分布式任务执行过程中，如果某些 TiDB 节点下线后，分布式任务将随机分配子任务到可用节点上，此时无法通过 `tidb_service_scope` 来动态均衡子任务的分配。
```
Comment (Frank945946) [inline tidb-distributed-execution-framework.md:83]: @ywqzzy 是分配到剩余的tidb_service_scope= 'Background' 的 TiDB node 对吧。如果用户只有 1 个 Background 节点，且这个节点挂了，那这个任务是不是就挂了，因为没有冗余的被标记为 Background 的节点

File: tidb-distributed-execution-framework.md [hunk 4]
```diff
@@ -78,9 +78,10 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
...
+    > - 在分布式任务执行过程中，如果子任务均下发完毕，此时无法通过 `tidb_service_scope` 配置新节点执行该任务。
```
Comment (Frank945946) [inline tidb-distributed-execution-framework.md:84]: @ywqzzy 有比较直观的方式可以判断子任务已经下发了么？如果没有，就建议直接说当前不支持任务已经提交后，将任务 rebalance 到新添加的且 `tidb_service_scope` 标记为 Background的 TiDB 节点。

File: tidb-distributed-execution-framework.md (hunk 3 above)
Comment (ywqzzy) [reply tidb-distributed-execution-framework.md:83]: 如果只有一个 background 节点，还剩一个默认节点，就会转移到默认节点执行。

File: tidb-distributed-execution-framework.md (hunk 4 above)
Comment (ywqzzy) [reply tidb-distributed-execution-framework.md:84]: 目前没有特别直观的方式，需要查 tidb 的 tidb_background_subtasks 表，或者看监控信息。

Comment (Frank945946) [reply tidb-distributed-execution-framework.md:84]: 那建议这期就说任务提交后，不支持 rebalance 到新添加的且 tidb_service_scope= 'Background' 的 TiDB 节点。等后面支持后，再去掉。

Comment (Frank945946) [inline tidb-distributed-execution-framework.md:84]:
```suggestion
    > - 在分布式任务执行过程中，此时无法通过 `tidb_service_scope` 配置新节点执行该任务。
```

**Review by Frank945946 — APPROVED** [review]

**Review by ywqzzy — APPROVED** [review]

File: tidb-distributed-execution-framework.md (hunk 3 above)
Comment (qiancai) [inline tidb-distributed-execution-framework.md:83]:
```suggestion
    > - 在分布式任务执行过程中，如果某些 TiDB 节点下线，分布式任务将随机分配子任务到其他可用的 TiDB 节点上，此时无法通过 `tidb_service_scope` 来动态控制子任务的分配。
```

File: tidb-distributed-execution-framework.md [hunk 5]
```diff
@@ -78,9 +78,10 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
 
 3. 从 v7.4.0 开始，你可以根据实际需求，调整执行后端任务的 TiDB 节点数量，在部署 TiDB 后为每一个 TiDB 节点设置 Instance 级别系统变量 [`tidb_service_scope`](/system-variables.md#tidb_service_scope-从-v740-版本开始引入)。`tidb_service_scope` 设置为 `background` 时，TiDB 节点可执行后端任务。`tidb_service_scope` 设置为默认值 "" 时，TiDB 节点不可执行后端任务。如果所有节点均未配置 `tidb_service_scope`，框架将调度所有节点执行后端任务。
 
-    > **警告：**
+    > **注意：**
     >
-    > `tidb_service_scope` 目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+    > - 在分布式任务执行过程中，如果某些 TiDB 节点下线后，分布式任务将随机分配子任务到可用节点上，此时无法通过 `tidb_service_scope` 来动态均衡子任务的分配。
+    > - 在分布式任务执行过程中，不支持通过 `tidb_service_scope` 配置新节点执行该任务。
```
Comment (qiancai) [inline tidb-distributed-execution-framework.md:84]: 这是是指配置 `tidb_service_scope` 会报错吗，还是暂时不生效

Comment (ywqzzy) [reply tidb-distributed-execution-framework.md:84]: 不生效

Comment (qiancai) [inline tidb-distributed-execution-framework.md:84]: 那可以修改成这样吗：

```suggestion
    > - 在分布式任务执行过程中，修改 `tidb_service_scope` 的配不会对当前任务生效，会从下次任务开始生效。
```

Comment (ywqzzy) [reply tidb-distributed-execution-framework.md:84]: LGTM

**Review by qiancai — APPROVED** [review]

File: tidb-distributed-execution-framework.md [hunk 6]
```diff
@@ -78,9 +78,10 @@ TiDB 采用计算存储分离架构，具有出色的扩展性和弹性的扩缩
 
 3. 从 v7.4.0 开始，你可以根据实际需求，调整执行后端任务的 TiDB 节点数量，在部署 TiDB 后为每一个 TiDB 节点设置 Instance 级别系统变量 [`tidb_service_scope`](/system-variables.md#tidb_service_scope-从-v740-版本开始引入)。`tidb_service_scope` 设置为 `background` 时，TiDB 节点可执行后端任务。`tidb_service_scope` 设置为默认值 "" 时，TiDB 节点不可执行后端任务。如果所有节点均未配置 `tidb_service_scope`，框架将调度所有节点执行后端任务。
 
-    > **警告：**
+    > **注意：**
     >
-    > `tidb_service_scope` 目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+    > - 在分布式任务执行过程中，如果某些 TiDB 节点下线，分布式任务将随机分配子任务到其他可用的 TiDB 节点上，此时无法通过 `tidb_service_scope` 来动态控制子任务的分配。
```
Comment (hfxsd) [inline tidb-distributed-execution-framework.md:83]:
```suggestion
    > - 在分布式任务执行过程中，如果某些 TiDB 节点下线，分布式任务将随机分配子任务到其他可用的 TiDB 节点上，此时无法通过 `tidb_service_scope` 来动态控制子任务的分配。 
```

**Pull request merged**